project2/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+
+
+
+
+
+
+<body>
+
+
+
+
+
+    <div class="card">
+        <img src="Healthy-Garlic-Shrimp-Pasta-with-Lemon.jpg" alt="">
+
+        <h1>Garlic Shrimp Pasta
+        </h1>
+
+
+        <ul class="menu">
+            <li><a href="#fas fa-shopping-cart"><i class="fas fa-shopping-cart"></i></a></li>
+            <li><a href="#fas fa-utensil-spoon"><i class="fas fa-utensil-spoon"></i></a></li>
+            <li><a href="#fas fa-clock"><i class="fas fa-clock"></i></a></li>
+        </ul>
+
+
+
+        <section id="fas fa-shopping-cart">
```

Comment:
An element can have only one `id`. Here you've repeated the classnames used for the icons. Same note on line 62 and 89 below.
```suggestion
       <section id="instructions">
```